src/components/EditCoursePage/index.jsx

```diff
@@ -154,8 +154,21 @@ class EditCoursePage extends React.Component {
     } = this.props;
     const { startedFetching } = this.state;
 
+    const courseStatuses = [];
     const courseInReview = course_runs && course_runs.some(courseRun =>
       IN_REVIEW_STATUS.includes(courseRun.status));
+    courseStatuses.push(courseInReview && IN_REVIEW_STATUS[0]);
+    const coursePublished = course_runs && course_runs.some(courseRun =>
+      PUBLISHED === courseRun.status) && PUBLISHED;
+    courseStatuses.push(coursePublished);
+    const courseReviewed = course_runs && !coursePublished && course_runs.some(courseRun =>
+      REVIEWED === courseRun.status) && REVIEWED;
+    courseStatuses.push(courseReviewed);
+    const courseUnpublished = course_runs && !coursePublished && !courseInReview &&
+      !courseReviewed && course_runs.some(courseRun => UNPUBLISHED === courseRun.status) &&
+      UNPUBLISHED;
+    courseStatuses.push(courseUnpublished);
```

Comment:
These statuses aren't mutually exclusive? Why the list pushing instead of an if/else situation?

Reply:
They are not mutually exclusive for a Course. One scenario is the course is live and has a different course run in review so it has both the live and in review statuses. Eventually, we will also support having an `unsaved edits` status (or something similar) that will also add to this list.